expyfun/stimuli/tests/test_tracker.py

```diff
@@ -164,12 +163,11 @@ def test_tracker_dealer():
     assert_raises(RuntimeError, dealer_ud.get_trial)
 
     # test array-like indexing
-    dealer_ud[0]
-    dealer_ud[:]
-    dealer_ud[[1, 0, 1]]
-    [d for d in dealer_ud]
+    dealer_ud.trackers[0]
+    dealer_ud.trackers[:]
+    dealer_ud.trackers[[1, 0, 1]]
+    [d for d in dealer_ud.trackers]
     dealer_ud.shape
```

Comment:
Does having the shape property return the shape of the trackers still make sense?